__tests__/app.test.js

```diff
@@ -357,3 +356,43 @@ describe("GET /api/users", () => {
   });
 });
 
+describe("GET /api/articles (sorting queries)", () => {
+  test("should default sort by created_at and default order by desc", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(13);
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+      });
+  });
```

Comment:
Nice, but this should be in the original testing for the endpoint and isnt a new behaviour